src/clusterfuzz/_internal/system/environment.py

```diff
@@ -1141,3 +1141,36 @@ def wrapper(*args, **kwargs):
     return None
 
   return wrapper
+
+
+def is_testcase_deprecated(platform_id=None):
+  """Whether or not the device or branch is deprecated."""
+  if not is_android(platform_id.upper()):
+    return False
+
+  if is_android_cuttlefish(platform_id.upper()):
```

Comment:
This would be much cleaner as
```
if is_android and not is_android_cuttlefish(platform_id.upper()):
    from clusterfuzz._internal.platforms import android

    return android.util.is_testcase_deprecated(platform_id)
return False
```

Reply:
Done